CHANGELOG.md

```diff
@@ -9,108 +9,37 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [12.5.0]
 
 ## [12.4.0]
```

Comment:
This first `## [12.4.0]` should not be here

Reply:
Thanks @danjm, now its addressed after your comment.